src/components/Button/README.md

```diff
@@ -187,6 +187,18 @@ Use to render a buttons that look like links where matching the surrounding text
 />
 ```
 
+### Plain destructive button
+
+<!-- example-for: online-store-ui -->
+
+Use to render a buttons that look like links where matching the surrounding text color is visually more appropriate. For example in the InlineError component.
+
+```jsx
```

Comment:
Let's also add a new example to the `Card` component `README` that shows a destructive card action, since that's the intended/main use case 👍

Reply:
@calumptrck You'll also need to update the action type of `Card.action` from `DisableableAction` to `DestructibleAction` in order to make a card action destructible.